src/test/java/com/hpe/adm/octane/ideplugins/integrationtests/IntegrationTestBase.java

```diff
@@ -582,4 +643,52 @@ public void deleteBacklogItems() {
                 octane.entityList("work_items").delete().query(workItemsQuery.build()).execute();
         }
     }
+
+    public EntityModel testSearch(String searchField, String query) {
```

Comment:
Why is this in the base and not in the search test class?

Reply:
it is supposed to be used by both requirements unit test and search functionality unit tests. I will rename the function to "search" since testSearch is indeed bad naming.